src/client/styles/style-v2.scss

```diff
@@ -169,8 +169,8 @@ dl dd.multi-item-list,
 .nypl-results-sorting-controls.style-2 {
 
   @include results-sorting-controls(
-    $border: 0,
-    $text-transform: uppercase);
```

Comment:
Because of the updates with toolkit v0.1.24.2, this variable will point to a value that no longer exists.